recipes/roounfold/recipe.yaml

```
@@ -0,0 +1,87 @@
+context:
+  name: RooUnfold
+  version: "3.0.5"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://gitlab.cern.ch/RooUnfold/RooUnfold/-/archive/${{ version }}/${{ name }}-${{ version }}.tar.gz
+  sha256: e57d846efcaff831dfeaad528fa9ebc60fe3e09165decbbc261592c23851d40a
+  patches:
+    - patches/cmake-minimum-3.5.patch
+
+build:
+  number: 0
+  script:
+    - cmake ${CMAKE_ARGS}  -S . -B build
+    - cmake --build build --clean-first --parallel="${CPU_COUNT}"
+    - cmake --install build
+    # NOTE: we must manually set SP_DIR since it is not set.
+    # Can be removed once https://github.com/prefix-dev/rattler-build/issues/1733 is fixed.
+    - if: unix
+      then:
+        - export SP_DIR=$(python -c "import site; print(site.getsitepackages()[0])")
+    - if: osx
+      then:
+        - export SP_DIR=$(python -c "import site; print(site.getsitepackages()[0])")
```

```suggestion
    - if: unix
      then:
        - export SP_DIR=$(python -c "import site; print(site.getsitepackages()[0])")
```

`unix` should include `osx`. Were you using this to debug?